client/components/AddEventForm.jsx

```diff
@@ -60,14 +80,28 @@ export class Component extends React.Component {
                 <div>
                     <Field name="dates.start"
                            component={DayPickerInput}
-                           withTime={!this.state.isFullDay}/>
+                           withTime={true}/>
                     &nbsp;to&nbsp;
                     <Field name="dates.end"
+                           defaultDate={this.theDayAfterStartingDate()}
                            component={DayPickerInput}
-                           withTime={!this.state.isFullDay}/>
+                           withTime={true}/>
+                </div>
+                <div>
+                    <label htmlFor="repeat">Repeat ...</label>
+                    <input
+                        name="doesRepeat"
+                        type="checkbox"
+                        value={true}
+                        checked={this.state.doesRepeat}
+                        onChange={this.handleDoesRepeatChange.bind(this)}/>
```

Comment:
I noticed that half of React developers bind methods in constructor but I personally prefer to bind them where I use them. Like that renaming is easier, definitions are less ambiguous and constructor keeps its role of "state" setter.
Why do you prefer to have bindings in constructor ?
I would prefer to allow arrow function syntax for class methods